### DISCUSSION TITLE
Add progressive pace caret

### DISCUSSION THREAD

**Discussion by precompute** [monkeytypegame/monkeytype#1236] [Ideas] [answered]
The speed of the caret would be PB + x, so it's always challenging.

**Comment by Miodec** [comment, marked as answer]
So, in my opinion a better version of this is the average pace caret. Beating your pb is rare, and its much better to try to beat your average to steadily approach your pb.

**Reply by precompute** [reply]
I saw that the average only cosiders the last 10 tests.  That's great.

I think a progressive caret could make it really challenging and help users progress quicker by subverting the unconscious bias that only allows one to outstrip the caret by so much (because it's always visible and moving, and close to the live caret).   Beating the average caret every time would be an expectation and wouldn't give a solid indication of progress, but even keeping up with the progressive caret would be an achievement.  It would be really helpful to slow typists, and to users learning a new layout.

I don't have anything to back this up except a hunch.

**Reply by frogfrogfrogfrogfrog** [reply]
The progressive pb seems too hard of a goal to reach in my opinion. Also it seems meh since your focusing on speed rather than accuracy when your trying to beat this progressive pb. When your beating your average you'd want your accuracy to be one of the main factors in why you got a new average pb.
